contracts/src/models/config.cairo

```diff
@@ -344,6 +344,16 @@ pub struct BankConfig {
     lp_fee_denom: u128,
 }
 
+#[derive(Copy, Drop, Serde)]
+#[dojo::model]
+pub struct BuildingGeneralConfig {
+    #[key]
+    config_id: ID,
+    // cost scale percent
+    base_cost_percent_increase: u16,
```

Comment:
remove comment and change the name to be more explicit, we don't want comments to be everywhere if it can be explained through the name imo

Reply:
i dont actually know what a better name would be. ive renamed it like 5 times

Reply:
`increase_per_extra_building_percent` ? Honestly I prefer explicit and long than short haha

Reply:
`quadratic_increase_per_extra_building_percent`

Reply:
it doesnt fit into a felt. long names wont fit into a felt

Reply:
Forgot dojo had those limitations

Reply:
31 characters at most